etna/records/blocks.py

```diff
@@ -61,7 +61,7 @@ def bulk_to_python(self, values):
         if not values:
             return []
         records_by_id = {
-            r.iaid: r for r in self.target_model.api.fetch_all(iaids=values)[1]
+            r.iaid: r for r in self.target_model.api.fetch_all(metadata_ids=values)[1]
```

Comment:
Looks like we're still taking the `iaid` value of the records here as the canonical id.

Reply:
Going with discussions.

Reply:
On this @TNA-Allan. The solution here will currently only work for records with an `.iaid` value. If the point of these changes it to accommodate records without an `iaid`, then shouldn't we use a different attribute here?

Reply:
The current api response and attribute have the same name i.e. `iaid` until now, no other source is formalised.